src/main/java/org/fogbowcloud/sebal/SEBALHelper.java

```diff
@@ -60,83 +45,78 @@
 import org.opengis.referencing.operation.TransformException;
 
```

Comment:
SEBALHelper is not a good name. It would be better to have XPTOUtil classes that group related stuff together.